Azure/azure-cli
# [Sql db] - Add missing word in docs description

**Description**
Description of the cli command didn't read right.

**Testing Guide**
Just documentation changes


---

This checklist is used to make sure that common guidelines for a pull request are followed.
